<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatRoom</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#36393f;</string>
  </property>
  <widget class="QTextBrowser" name="Show_Chat">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>661</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #fff;
border-radius:15px;
color:#fff;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Text_Butten">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>495</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
background-color:#5865f2;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#5865f2;
}
</string>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Show_name">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>101</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #fff;
border-radius:15px;
color:#fff;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="credit_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:25px;
background-color:#fff;
color:#5865f2;
}

QPushButton:hover{
border-radius:20px;
background-color:#5865f2;
color:#fff;
}


</string>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Show_name_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>81</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2f3136;
color:#fff;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Main_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:20px;
background-color:#fff;
color:#5865f2;
}

QPushButton:hover{
border-radius:20px;
background-color:#fff;
color:#5865f2;
}

</string>
   </property>
   <property name="text">
    <string>MAIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Room1_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:25px;
background-color:#5865f2;
color:#fff;
}

QPushButton:hover{
border-radius:20px;
background-color:#fff;
color:#5865f2;
}</string>
   </property>
   <property name="text">
    <string>Room1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Room2_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:25px;
background-color:#5865f2;
color:#fff;
}

QPushButton:hover{
border-radius:20px;
background-color:#fff;
color:#5865f2;
}
</string>
   </property>
   <property name="text">
    <string>Room2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Room3_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:25px;
background-color:#5865f2;
color:#fff;
}

QPushButton:hover{
border-radius:20px;
background-color:#fff;
color:#5865f2;
}

</string>
   </property>
   <property name="text">
    <string>Room3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>485</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:25px;
background-color:#fff;
color:#5865f2;
}

QPushButton:hover{
border-radius:20px;
background-color:#5865f2;
color:#fff;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msg_Edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>495</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border:1px solid #fff;
border-radius:5px;
border-radius:15px;
background-color:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_Btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>495</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
background-color:#5865f2;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#5865f2;
}</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <zorder>Show_Chat</zorder>
  <zorder>Text_Butten</zorder>
  <zorder>Show_name</zorder>
  <zorder>Show_name_2</zorder>
  <zorder>Main_Btn</zorder>
  <zorder>Room1_Btn</zorder>
  <zorder>Room2_Btn</zorder>
  <zorder>Room3_Btn</zorder>
  <zorder>Exit_Btn</zorder>
  <zorder>msg_Edit</zorder>
  <zorder>credit_Btn</zorder>
  <zorder>Clear_Btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
